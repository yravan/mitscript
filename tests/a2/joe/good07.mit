// test builtin functions print and intcast

print("\"");
print("\\");
print("\\\\");

print("\ttab");
print("\\ttab");
print("\\\ttab");

print("\nnewline");
print("\\nnewline");
print("\\\nnewline");

x = intcast("00002");
print(x);

x = intcast("-1");
print(x);

