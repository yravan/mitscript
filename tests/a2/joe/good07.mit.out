"
\
\\
	tab
\ttab
\	tab

newline
\nnewline
\
newline
2
-1
